src/managers/GuildEmojiRoleManager.js

```diff
@@ -71,7 +71,10 @@ class GuildEmojiRoleManager {
   remove(roleOrRoles) {
     if (roleOrRoles instanceof Collection) return this.remove(roleOrRoles.keyArray());
     if (!Array.isArray(roleOrRoles)) return this.remove([roleOrRoles]);
-    roleOrRoles = roleOrRoles.map(r => this.guild.roles.resolveID(r));
+    roleOrRoles = roleOrRoles.map(role => {
+      const resolvedRole = this.guild.roles.resolve(role);
+      return resolvedRole && resolvedRole.id;
+    });
```

Comment:
Any reason for this change?

Reply:
it was changed like this to be consistent with the `resolve` usage, there were some usages of `resolve` and `resolveID`